UI/window-projector.cpp

```diff
@@ -148,6 +154,38 @@ void OBSProjector::SetHideCursor()
 		setCursor(Qt::ArrowCursor);
 }
 
+void OBSProjector::SetHideFrame(bool hideFrame)
+{
+	if (isFullScreen())
+		return;
+
+	// Calculate the current content position.
+	QRect contentBox = geometry();
+
+	// Calculate the current window geometry position.
+	QPoint contentPos = pos();
+
+	// Only update window if setting changed.
+	if (this->hideFrame != hideFrame) {
```

Comment:
It would be less indents to return early like this. And, calling `pos()` and `geometry()` should be later than checking `hideFrame`.
```c
	if (this->hideFrame == hideFrame)
		return;
```